### ISSUE TITLE
Command '['hostname -I']' returned non-zero exit status 1.

### ISSUE THREAD

**Issue by linbingkong** [KimMeen/Time-LLM#155]
train_loader, vali_loader, test_loader, model, model_optim, scheduler = accelerator.prepare(train_loader, vali_loader, test_loader,model, model_optim, scheduler) raise error:Command '['hostname -I']' returned non-zero exit status 1.

**Comment by linbingkong** [comment]
![image](https://github.com/user-attachments/assets/16d2a1e6-5627-44d2-af1c-3fe6d599d89b)
![image](https://github.com/user-attachments/assets/4043a11f-02c1-4cbf-b904-5a1b6985e74a)

**Comment by Dclpwos** [comment]
I encounter the same problem！

**Comment by guoqian-01** [comment]
I'm having the same problem, I didn't have it when I ran it in October and before, have you solved it yet?
